lib/fhir/query-utils.js

```diff
@@ -450,40 +459,85 @@ module.exports = (mongo) => {
      *                             chained parameters
      */
     genChainedParamQuery: (value, baseParam, chainedParam, chainedModule, callback) => {
```

Comment:
Please add to the comment for how the function signature has changed (allowing arrays).